.github/workflows/go-standards.yml

```diff
@@ -0,0 +1,62 @@
+# This workflow will build, test, and lint a Golang project.
+# For more information see:
+# - https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+# - https://golangci-lint.run/usage/install/#github-action
+
+name: Go Standards (Build, Test & Lint)
+
+on:
+  push:
+    branches: ["main", "ci", "test"]
+  pull_request:
+    branches: ["main", "ci", "test"]
+
+permissions:
+  contents: read
+
+jobs:
+  build-test-lint:
+    # This job runs on the latest Ubuntu environment
+    runs-on: ubuntu-latest
+
+    steps:
+      # Step 1: Check out the repository
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      # Step 2: Set up Go environment
+      - name: Set up Go v1.22.3
```

Comment:
I wouldn't use the version in the comment. The next time you will update the version node, you might forget to update this.

Reply:
But, you can add a new steps and use a run cmd to launch `go version` so you have it in the GHA logs